Gemfile

```diff
@@ -26,7 +26,11 @@ gem 'turbolinks', '~> 5'
 # Build JSON APIs with ease. Read more: https://github.com/rails/jbuilder
 gem 'jbuilder', '~> 2.5'
 
+gem 'bootstrap-sass'
```

Comment:
Could you install all these stuff (except `slim-rails`) using `yarn`? With yarn, we'll be able to use it with the webpack which is better than default rails assets pipeline.